cmd/plugin/install.go

```diff
@@ -25,22 +27,20 @@ func NewInstallCommand(serverConfig *config.ServerConfig) *cobra.Command {
 		RunE:    install.RunE,
 		PreRunE: install.PreRunE,
 	}
+	cmd.PersistentFlags().StringVarP(&install.configFilePath, "config", "c", install.configFilePath, "File path for server configuration")
 	return cmd
 }
 
 func (i *installCommand) PreRunE(_ *cobra.Command, _ []string) error {
-	i.logger = logger.NewClientLogger(i.serverConfig.Log)
+	c, err := config.LoadServerConfig(i.configFilePath)
+	if err != nil {
+		return err
+	}
+	*i.serverConfig = *c
```

Comment:
Why are we using pointer to a pointer for setting value. ?
LoadServerConfig returns a pointer to `ServerConfig` which we can set directly in struct.

```go
	i.serverConfig = c
```